python/ray/dag/compiled_dag_node.py

```diff
@@ -670,6 +683,12 @@ def _get_or_create_local_actor_handle():
 
         self._dag_creator_actor = _get_or_create_local_actor_handle()
 
+    def increment_max_finished_execution_index(self) -> None:
```

Comment:
Do we really need this method? It just has a single statement

Reply:
I don't want to increment it by acessing internal attribute, lmk if you have other alternative